backends/cuda/ceed-cuda-compile.cpp

```diff
@@ -45,10 +52,23 @@ static int CeedCompileCore_Cuda(Ceed ceed, const char *source, const bool throw_
   struct cudaDeviceProp prop;
   Ceed_Cuda            *ceed_data;
 
+
   cudaFree(0);  // Make sure a Context exists for nvrtc
 
   std::ostringstream code;
 
+  bool usingClang;
+  CeedCallBackend(CeedGetIsClang(ceed, &usingClang));
```

Comment:
```suggestion
  bool using_clang;
  CeedCallBackend(CeedGetIsClang(ceed, &using_clang));
```
note - snake_case is the variable convention. Also, one blank line after variable declaration blocks